apps/guardian-ui/src/AppContext.tsx

```diff
@@ -7,50 +7,20 @@ import React, {
 } from 'react';
 import { GuardianApi } from './GuardianApi';
 import { formatApiErrorMessage } from './utils/api';
+import { ServerStatus } from './types';
 
-const LOCAL_STORAGE_KEY = 'guardian-ui-state';
-
-function makeInitialState(loadFromStorage = true): AppState {
-  let storageState: Partial<AppState> = {};
-  if (loadFromStorage) {
-    try {
-      const storageJson = localStorage.getItem(LOCAL_STORAGE_KEY);
-      if (storageJson) {
-        storageState = JSON.parse(storageJson);
-      }
-    } catch (err) {
-      console.warn('Encountered error while fetching storage state', err);
-    }
-  }
-
-  return {
-    experience: 'Setup',
-    ...storageState,
-  };
-}
-
-const initialState = makeInitialState();
-
-interface AppState {
-  experience: 'Setup' | 'Admin';
-}
-
-interface ApiState {
-  connected: boolean;
-  error: string;
-}
+type AppState = 'Loading' | 'Setup' | 'Admin' | 'Error';
```

Comment:
This is kind of a nomenclature nitpick, but `AppState` tends to be an object representing the full state of something. I would consider this more of a `status` since it's a string. I think that would provide some improved readability to call it a status, since it was previously an object, and setup continues to use an object for its `state` variable and `SetupState` type as well.